cluster-autoscaler/cloudprovider/aws/aws_cloud_provider.go

```diff
@@ -227,6 +228,11 @@ func (asg *Asg) Nodes() ([]string, error) {
 	return asg.awsManager.GetAsgNodes(asg)
 }
 
+// TemplateNodeInfo returns a node template for this node group.
```

Comment:
Perhaps what I wanted was https://github.com/kubernetes/autoscaler/pull/44/files#diff-dda4cc04e6ebb992f7300f51a237d75dR80? :)
You are going to create a template node info from e.g. an ASG/Launch configuration spec for AWS so that it can be used for estimating the capacity after a possible scaling out of nodes, right?